docs/content/latest/releases/earlier-releases/v2.0.9.md

```diff
@@ -70,16 +67,15 @@ docker pull yugabytedb/yugabyte:2.0.9.0-b13
 * Delete provisional records RocksDB SSTable files. [#2186](https://github.com/yugabyte/yugabyte-db/issues/2186) 
 
 ## Platform
+
 * [YW] Change Intro dialog to only appear on login. Add checkbox in Intro dialog for permanently hiding modal. [#3138](https://github.com/yugabyte/yugabyte-db/issues/3138)
 * [Platform] Fix console errors in UI code [#2790](https://github.com/yugabyte/yugabyte-db/issues/2790)
 * [YW] Fix backup Universe from changing to undefined url
 * [Platform] Task history not reflecting when a encryption is enabled/disabled/rotated [#2929](https://github.com/yugabyte/yugabyte-db/issues/2929)
 * Add per node metrics for health. [#2894](https://github.com/yugabyte/yugabyte-db/issues/2894)
-* Implement RBAC in Yugaware. [#2793](https://github.com/yugabyte/yugabyte-db/issues/2793)
+* Implement RBAC in YugaWare. [#2793](https://github.com/yugabyte/yugabyte-db/issues/2793)
```

Comment:
```suggestion
* Implement RBAC in Yugabyte Platform(YugaWare). [#2793](https://github.com/yugabyte/yugabyte-db/issues/2793)
```
Just suggesting the same way based on how you have replaced it at some of the other places.

Reply:
As at that release (2.0.9) product was still called YugaWare so I left it